sample-apps/donet-sample-app/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+COPY . ./
+RUN dotnet publish donet-sample-app/*.csproj -c Release -o out
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
+WORKDIR /app
+EXPOSE 8080
+COPY --from=build-env /app/out .
+ENTRYPOINT ["dotnet", "donet-sample-app.dll"]
+
+[
```

Comment:
was the rest of this file added by mistake here?

Reply:
Yeah, fixed in new PR